docs/recipes/babel.md

```diff
@@ -101,6 +101,31 @@ Note that this *does not* stop AVA from compiling your test files using Babel.
 
 If you want, you can disable the preset in your project's Babel configuration.
 
+## Change Power Assert's patterns
+
+If you prefer to name your `t` argument something else, or access `assert` with destructuring assignment, you need to tell [`power-assert`](https://github.com/power-assert-js/power-assert) which patterns to look for.
+
+**`package.json`:**
+
+```json
+{
+	"ava": {
+		"babel": {
+			"testOptions": {
+				"plugins": [
+					["babel-plugin-espower", {
+						"patterns": [
+							"assert(value, [message])",
+							"test.assert(value, [message])"
```

Comment:
This would be `t.assert`.

Reply:
My intention was to show how to deal with an argument name other than `t`, and I thought `test` ought to be the most clear example. Perhaps the example should include `t.assert` as well.

Reply:
It's this confusion that makes me think we should support `assert` usage by default.

Reply:
I was confused by `test.assert` too. I think you should use a totally unrelated name like `foo.assert`.